src/locale/de.json

```
@@ -118,8 +121,11 @@
         "C4": "C4 - Deponie",
         "D": "D - Wiederverwendung"
       },
-      "totalCO2Equivalents": "(Bisherige) CO2 Emissionen des Produkts",
-      "years": "Jahre"
+      "timeunit": {
+        "month": "{count, plural, =0 {unter einem Monat} =1 {einen Monat} other {# Monate}}",
+        "year": "{count, plural, =0 {unter einem Jahr} =1 {ein Jahr} other {# Jahre}}"
```

I would prefer to write '1 Monat' and '1 Jahr' instead of 'ein'/'einen' for consistency. Otherwise the grammatically correct way would be to write the numbers out up to 12. 

```suggestion
        "month": "{count, plural, =0 {unter einem Monat} =1 {# Monat} other {# Monate}}",
        "year": "{count, plural, =0 {unter einem Jahr} =1 {# Jahr} other {# Jahre}}"
```